flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/detail/job-checkpoints-detail.component.ts

```diff
@@ -57,9 +60,23 @@ export class JobCheckpointsDetailComponent implements OnInit {
   refresh() {
```

Comment:
Hi @gm7y8 
could you try to replace the refresh function with the codes below

```ts

  refresh() {
    this.isLoading = true;
    if (this.jobDetail && this.jobDetail.jid) {
      forkJoin([
        this.jobService.loadCheckpointConfig(this.jobDetail.jid),
        this.jobService.loadCheckpointDetails(this.jobDetail.jid, this.checkPoint.id)
      ]).subscribe(
        ([config, detail]) => {
          this.checkPointConfig = config;
          this.checkPointDetail = detail;
          if (this.checkPointDetail.checkpoint_type === 'CHECKPOINT') {
            if (this.checkPointConfig.unaligned_checkpoints) {
              this.checkPointType = 'unaligned checkpoint';
            } else {
              this.checkPointType = 'aligned checkpoint';
            }
          } else if (this.checkPointDetail.checkpoint_type === 'SYNC_SAVEPOINT') {
            this.checkPointType = 'savepoint on cancel';
          } else if (this.checkPointDetail.checkpoint_type === 'SAVEPOINT') {
            this.checkPointType = 'savepoint';
          } else {
            this.checkPointType = '-';
          }
          this.isLoading = false;
          this.cdr.markForCheck();
        },
        () => {
          this.isLoading = false;
          this.cdr.markForCheck();
        }
      );
    }
  }

```

Reply:
done!